script/check_publish.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+set -e
+
+# This script checks to make sure that each of the plugins *could* be published.
+# It doesn't actually publish anything.
+
+# So that users can run this script from anywhere and it will work as expected.
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null && pwd)"
+REPO_DIR="$(dirname "$SCRIPT_DIR")"
+
+source "$SCRIPT_DIR/common.sh"
+
+function check_publish() {
```

Comment:
Do we need to update any documentation (such as contributing.md) to point to the existence of such a script? or is it forced by Cirrus?

Reply:
No, not forced by Cirrus.  I'll add a line in CONTRIBUTING.md